### PR TITLE
PyDMShellCommand bash option

### PR DESCRIPTION
**Pull request by nstelter-slac**
allow for Bash features in PyDMShellCommand cmd such as command chaining ("command1; command2")

also add example that shows 2 alternative ways to use Bash through PyDMShellCommand widget.

### CODE REVIEW

**Comment by klauer** [comment]
A few things:
* It's not really bash though, at least not necessarily:
```
In [2]: subprocess.call('echo $0', shell=True)
/bin/sh
```

image

[ref](https://docs.python.org/3/library/subprocess.html#subprocess.Popen)
* It's perhaps also worth a warning _somewhere_ that globs, variables, etc, will be expanded.
* I'm personally a fan of distributing small shell scripts alongside PyDM screens as they can be used externally, though I do see the utility here.
* For the designer, it'd be really nice if the shell command could have its own multi-line text edit widget so that you specifically _don't_ have to `chain; commands; together; in` an unreadable fashion

File: examples/shell_command/shell_command_full_shell.ui [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+   <item>
+    <widget class="PyDMShellCommand" name="PyDMShellCommand">
+     <property name="toolTip">
+      <string/>
+     </property>
+     <property name="whatsThis">
+      <string/>
+     </property>
+     <property name="text">
+      <string>runCmdsInFullShell Option Enabled</string>
```
Comment (jbellister-slac) [inline examples/shell_command/shell_command_full_shell.ui:42]: This first shell command should have the new option set to true so it works properly on load

File: examples/shell_command/shell_command_full_shell.ui [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+     </property>
+     <property name="allowMultipleExecutions" stdset="0">
+      <bool>false</bool>
+     </property>
+     <property name="titles" stdset="0">
+      <stringlist/>
+     </property>
+     <property name="commands" stdset="0">
+      <stringlist>
+       <string>echo Fist; echo Second</string>
```
Comment (jbellister-slac) [inline examples/shell_command/shell_command_full_shell.ui:79]:
```suggestion
       <string>echo First; echo Second</string>
```

File: pydm/widgets/shell_command.py [hunk 3]
```diff
@@ -126,6 +129,29 @@ def showConfirmDialog(self, value: bool) -> None:
         if self._show_confirm_dialog != value:
             self._show_confirm_dialog = value
 
+    @Property(bool)
+    def runCmdsInFullShell(self) -> bool:
```
Comment (jbellister-slac) [inline pydm/widgets/shell_command.py:133]: Can probably spell out the full word commands for the new property just to match the other properties in this widget and the style of the widget property names in general

**Review by jbellister-slac — COMMENTED** [review]
Looks good, thanks! I like the new example, just a few minor comments below

**Review by jbellister-slac — APPROVED** [review]

**Pull request merged**